### PR TITLE
osu!supporter pages image fixes

### PR DESCRIPTION
**Pull request by EpicSupermind**
also some http:// to https:// and grammar fixes

### CODE REVIEW

File: wiki/osu!supporter/en.md [hunk 1]
```diff
@@ -1,41 +1,39 @@
-![osu!supporter tag](Supporter tag.png "osu!supporter tag")
+# osu!supporter
 
-Introducing
-===========
+![osu!supporter tag](Supporter_tag.png "osu!supporter tag")
```
Comment (MegaApplePi) [inline wiki/osu!supporter/en.md]: Delete the image and replace with this link `osu!supporter tag`

File: wiki/osu!supporter/en.md [hunk 2]
```diff
@@ -1,41 +1,39 @@
...
 
-<img src="Osu-support.png" title=" osu!supporter link on main page." alt=" osu!supporter link on main page." width="200" />
+## Introducing
```
Comment (MegaApplePi) [inline wiki/osu!supporter/en.md]: Remove this heading.
This declares that this is the _lead section_ but the first paragraph of all articles are always implied to be the _lead section_, so this is redundant and should be removed.
It also doesn't make much sense anyways.

File: wiki/osu!supporter/en.md [hunk 3]
```diff
@@ -1,41 +1,37 @@
...
--   [Skinning](Skinning "wikilink"): Some additional skin elements can be used, such as the [main menu](http://osu.ppy.sh/forum/t/96949)'s background.
+-   [Chat Console](/wiki/Chat_Console): Bright yellow name others can see. However, you still see your username as white.
+-   [Beatmap listing](https://osu.ppy.sh/p/beatmaplist): Filter beatmap searches by played and unplayed maps.
+-   [Skinning](/wiki/Skinning): Some additional skin elements can be used, such as the [main menu](https://osu.ppy.sh/forum/t/96949)'s background.
     -   Specifically, `menu-background.jpg` (Main Menu); `triangle.jpg` (Triangles); `welcome.wav` (Welcome Sound); `seeya.wav` (Exiting Sound)
--   [Feature Requests](http://osu.ppy.sh/forum/4): Extra votes for feature requests (2 per month over usual 1 per month).
+-   [Feature Requests](https://osu.ppy.sh/forum/4): Extra votes for feature requests (2 per month over usual 1 per month).
 -   [Song Selection](/wiki/Interface): The ability to view beatmap's in your country / friends / mod-specific rankings in-game.
--   Your Profile: <http://osu.ppy.sh/u/>
+-   Your Profile: <https://osu.ppy.sh/u/>
```
Comment (TPGPL) [inline wiki/osu!supporter/en.md]: I'd change to Profile Advantages

Comment (EpicSupermind) [inline wiki/osu!supporter/en.md]: There's already a heading 
## Advantages of osu!supporter

**Pull request merged**